Source/RakNetSocket2_Berkley.cpp

```diff
@@ -35,8 +35,11 @@ void RNS2_Berkley::SetSocketOptions(size_t recvBufferSize, size_t sendBufferSize
 	r = setsockopt__( rns2Socket, SOL_SOCKET, SO_LINGER, ( char * ) & sock_opt, sizeof ( sock_opt ) );
 	// Do not assert, ignore failure
 
-
-
+#ifdef SO_NOSIGPIPE
+    sock_opt=1;
+    r = setsockopt__( rns2Socket, SOL_SOCKET, SO_NOSIGPIPE, ( char * ) & sock_opt, sizeof ( sock_opt ) );

```

Comment:
^ Being extra nitpicky here, but it would be best to keep indentation characters consistent with the rest of the method as to keep readability (it seems like either the file uses tabs and you used spaces, or the other way around.)